cmd/mattermost/commands/command.go

```diff
@@ -67,3 +90,71 @@ func moveCommandCmdF(command *cobra.Command, args []string) error {
 func moveCommand(a *app.App, team *model.Team, command *model.Command) *model.AppError {
 	return a.MoveCommand(team, command)
 }
+
+func createCommandCmdF(command *cobra.Command, args []string) error {
+	a, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer a.Shutdown()
+
+	team := getTeamFromTeamArg(a, args[0])
+	if team == nil {
+		return errors.New("unable to find team '" + args[0] + "'")
+	}
+
+	title, _ := command.Flags().GetString("title")
+	description, _ := command.Flags().GetString("description")
+	trigger, errt := command.Flags().GetString("trigger-word")
+	if errt != nil || trigger == "" {
+		return errors.New("atrigger word is required")
```

Comment:
You can simplify this kind of errors using the `MarkFlagRequired` for the mandatory flags. more info here: https://github.com/spf13/cobra#required-flags